verl/trainer/ppo/ray_trainer.py

```diff
@@ -562,6 +562,34 @@ def _create_dataloader(self):
             self.config.actor_rollout_ref.actor.optim.total_training_steps = total_training_steps
             self.config.critic.optim.total_training_steps = total_training_steps
 
+    def _maybe_log_train_generations_to_wandb(self, inputs, outputs, rewards):
+        """Log a table of training samples to wandb."""
+        # Only log if wandb is used
+        if 'wandb' not in self.config.trainer.logger:
```

Comment:
Create a book like config.trainer.logger.enable_wandb and check if it exists and is set to true instead of just checking if a key exists wothout checking it's value

Reply:
Oh, sorry, it was a debug code, I will remove it